package.json

```diff
@@ -19,22 +19,28 @@
     "lib"
   ],
   "main": "index.js",
+  "type": "module",
```

Comment:
For the long term, I think it would be better to change the files extensions to `.mjs` which would allow testing compatibility with CommonJS.

I didn't want to do that now so the diffs are reviewable. Maybe it can come in another PR?